qemu/tests/cfg/virtio_fs_sandbox.cfg

```
@@ -0,0 +1,76 @@
+- virtio_fs_sandbox:
+    no RHEL.6 RHEL.7 RHEL.8.0 RHEL.8.1
+    no Win2008 Win7
+    no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8.u0 Host_RHEL.m8.u1
+    type = virtio_fs_sandbox
+    virt_test_type = qemu
+    required_qemu = [4.2.0,)
+    s390, s390x:
+        required_qemu = [5.2.0,)
+        vm_mem_share = yes
+        pre_command_noncritical = yes
+        pre_command = "echo 3 > /proc/sys/vm/drop_caches"
+        setup_hugepages = yes
+        kvm_module_parameters = 'hpage=1'
+        expected_hugepage_size = 1024
+    kill_vm = yes
+    start_vm = no
+    filesystems = fs
+    fs_driver = virtio-fs
+    force_create_fs_source = no
+    fs_source_user_config = yes
+    remove_fs_source = yes
+    fs_target = 'myfs'
+    cmd_run_virtiofsd = '/usr/libexec/virtiofsd --socket-path %s --shared-dir %s'
+    fs_driver_props = {"queue-size": 1024}
+    mem = 4096
+    mem_devs = mem1
+    backend_mem_mem1 = memory-backend-file
+    mem-path_mem1 = /dev/shm
+    size_mem1 = 4G
+    use_mem_mem1 = no
+    share_mem = yes
+    !s390, s390x:
+        guest_numa_nodes = shm0
+        numa_memdev_shm0 = mem-mem1
+        numa_nodeid_shm0 = 0
+    io_timeout = 600
+    fs_dest = '/mnt/${fs_target}'
+    driver_name = viofs
+    Windows:
+        # install winfsp tool
+        i386, i686:
+            install_winfsp_path = 'C:\Program Files'
+            devcon_dirname = 'x86'
+        x86_64:
+            install_winfsp_path = 'C:\Program Files (x86)'
+            devcon_dirname = 'amd64'
+        install_winfsp_cmd = 'msiexec /i WIN_UTILS:\winfsp.msi /qn'
+        check_installed_cmd = 'dir "%s" |findstr /I winfsp'
+        viofs_svc_name = VirtioFsSvc
+        viofs_exe_path = C:\virtiofs.exe
+        viofs_exe_copy_cmd = xcopy %s C:\ /Y
+        viofs_sc_create_cmd = 'sc create ${viofs_svc_name} binpath=${viofs_exe_path} start=auto'
+        viofs_sc_create_cmd += ' depend="WinFsp.Launcher/VirtioFsDrv" DisplayName="Virtio FS Service"'
+        viofs_sc_start_cmd = 'sc start ${viofs_svc_name}'
+        viofs_sc_query_cmd = 'sc query ${viofs_svc_name}'
+        viofs_sc_delete_cmd = 'sc delete ${viofs_svc_name}'
```

```suggestion
        viofs_svc_name = VirtioFsSvc
```

Hello @xiagao , after I made a test, only `viofs_svc_name` is necessary. Since other parameters have been set in the virtio_fs_utils.py
I suggest deleting the unnecessary parameters.

---

Hi @nickzhq  I would like to keep them here to have a better understanding of the test.

---

> Hi @nickzhq I would like to keep them here to have a better understanding of the test.

That's okay to me